proto/metapb.proto

```diff
@@ -44,6 +51,8 @@ message Store {
     // Status address provides the HTTP service for external components
     string status_address = 7;
     string git_hash = 8;
+    // Indicates the location the store is at.
+    Location location = 9;
```

Comment:
I think there are 2 solution for this:
1 is configuring `group-id`s with stores, just like here. So leader can know the associated `group-id` for every store by asking PD.
2 is add `group-id` in `RaftMessage`, so leader can know followers' `group-id` after they send any messages to the leader.
PTAL @BusyJay